runtime/litmus/src/lib.rs

```diff
@@ -125,6 +126,7 @@ pub type Executive = frame_executive::Executive<
 	// it was reverse order before.
 	// See the comment before collation related pallets too.
 	AllPalletsWithSystem,
+	migration::AddNativeTokenForeignAssetTypeForAssetManager<Runtime>,
```

Comment:
I don't think we even need the migration, as no storage has ever been set on-chain in the asset manager. Please let me know if I misunderstand it.

Reply:
The problem is we miss the genesis build. We need to fix it somehow.

Reply:
But we missed the train of genesis build already 😂  the migration won't help the genesis build -- if you need a different state, we can directly change the pallet setting and/or send on-chain extrinsics, as the current state for it is pretty much empty anyway

Reply:
Correct!